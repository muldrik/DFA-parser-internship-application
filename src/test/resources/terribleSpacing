z=         10
u =   10  +          ( ( 1           * (5)))
t
  = 5*5
    + 6 * 6
    * (3+3)
    / 24
p = 0
if(1<2)while(p<0)p=p+1 t = p end end