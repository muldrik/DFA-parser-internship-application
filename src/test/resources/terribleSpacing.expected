z = 10
u = 10 + ((1 * (5)))
t = 5 * 5 + 6 * 6 * (3 + 3) / 24
t = p
